frontend/src/bundles/profile-cabinet/pages/styles.module.scss

```diff
@@ -17,7 +17,8 @@
     display: flex;
     flex-direction: column;
     gap: 26px;
-    max-width: 100%;
+    width: 560px;
```

Comment:
maybe move it to variables n assets and the same pages can use this width in similar potential bugs?